packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -37,116 +44,107 @@ export interface EmrCreateClusterProps {
    * The IAM role that will be assumed by the Amazon EMR service to access AWS resources on your behalf. A Role will be created if
    * one is not provided.
    *
-   * @default No serviceRole
+   * @default - None
    */
   readonly serviceRole?: iam.IRole;
 
   /**
    * A JSON string for selecting additional features.
    *
-   * @default No additionalInfo
+   * @default - None
    */
   readonly additionalInfo?: string;
 
   /**
    * A case-insensitive list of applications for Amazon EMR to install and configure when launching the cluster.
    *
-   * @default EMR selected default
+   * @default - EMR selected default
    */
   readonly applications?: EmrCreateCluster.ApplicationConfigProperty[];
 
   /**
    * An IAM role for automatic scaling policies. A Role will be created if one is not provided.
    *
-   * @default No autoScalingRole
+   * @default - None
    */
   readonly autoScalingRole?: iam.IRole;
 
   /**
    * A list of bootstrap actions to run before Hadoop starts on the cluster nodes.
    *
-   * @default No bootstrapActions
+   * @default - None
    */
   readonly bootstrapActions?: EmrCreateCluster.BootstrapActionConfigProperty[];
 
   /**
    * The list of configurations supplied for the EMR cluster you are creating.
    *
-   * @default No configurations
+   * @default - None
    */
   readonly configurations?: EmrCreateCluster.ConfigurationProperty[];
 
   /**
    * The ID of a custom Amazon EBS-backed Linux AMI.
    *
-   * @default No customAmiId
+   * @default - None
    */
   readonly customAmiId?: string;
 
   /**
-   * The size, in GiB, of the EBS root device volume of the Linux AMI that is used for each EC2 instance.
+   * The size of the EBS root device volume of the Linux AMI that is used for each EC2 instance.
    *
-   * @default EMR selected default
+   * @default - EMR selected default
    */
-  readonly ebsRootVolumeSize?: number;
+  readonly ebsRootVolumeSize?: cdk.Size;
 
   /**
    * Attributes for Kerberos configuration when Kerberos authentication is enabled using a security configuration.
    *
-   * @default No kerberosAttributes
+   * @default - None
    */
   readonly kerberosAttributes?: EmrCreateCluster.KerberosAttributesProperty;
 
   /**
    * The location in Amazon S3 to write the log files of the job flow.
    *
-   * @default No logUri
+   * @default - None
    */
   readonly logUri?: string;
```

Comment:
Something more strong using types in `aws-s3`?

Something like this - https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cloudtrail/lib/cloudtrail.ts#L415 ?